demos/TicTacToe/Classes/AppDelegate.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
```

Comment:
Instead of pragma once, use header guard:

https://google.github.io/styleguide/cppguide.html#The__define_Guard